base_layer/core/src/blockheader.rs

```diff
@@ -63,3 +63,20 @@ impl BlockHeader {
         unimplemented!();
     }
 }
+
+impl Hashable for BlockHeader {
+    fn hash(&self) -> Vec<u8> {
+        HashDigest::new()
+            .chain(self.version.to_le_bytes())
+            .chain(self.height.to_le_bytes())
+            .chain(self.prev_hash.as_bytes())
+            .chain(self.timestamp.timestamp().to_le_bytes())
+            .chain(self.output_mmr.as_bytes())
+            .chain(self.range_proof_mmr.as_bytes())
```

Comment:
Does this return the mmr root, or hash the whole thing?

Reply:
Just the root